antora-playbook.yml

```diff
@@ -136,6 +136,9 @@ content:
   - url: https://github.com/couchbaselabs/userprofile-couchbase-mobile-android
     branches: [standalone, query, sync]
     start_path: content
+  - url: https://github.com/couchbaselabs/OpenID_connect_tutorial
+    branches: [tutorial]
+    start_path: content
```

Comment:
There is no content directory in the repo atm.
Let me see if I can grab that from AK's tutorial branch, and put it in the 'labs one.  :+1: